app/serializers/person_serializer.rb

```diff
@@ -13,8 +13,6 @@ class PersonSerializer < ActiveModel::Serializer
     :gender_additional_information,
   )
 
```

Comment:
This deletion introduces new behaviour. It's now not possible to tell if a person should have an image property or not. I think we should therefore have a test to show that if we don't have a prison_number on the person, then /people/<id>/images behaves correctly (which I think means 404)

Reply:
This was discussed with @teneightfive, it complicates the FE logic, we preferred to simplify the logic removing it,
We might want to put it back in the future, so FE would need to retrieve the images only for the people that do not have yet URL,